AnalyticsApp/AnalyticsApp/ContentView.swift

```diff
@@ -6,30 +6,52 @@
 //
 
 import SwiftUI
+import Analytics
 
 struct ContentView: View {
     
     var body: some View {
         VStack {
             HStack {
                 CustomButton(title: "Track") {
-                    AnalyticsManager.shared.analytics?.track(name: "Track at \(Date())", properties: ["key": "value"], options: ["option": "value"])
+                    let option = RudderOptions()
+                    .addIntegration("SDK", isEnabled: true)
+                    .addIntegration("Segment", isEnabled: false)
```

Comment:
Please replace it with real values like `Amplitude` or `CleverTap`.
```suggestion
                    .addIntegration("Amplitude", isEnabled: true)
                    .addIntegration("CleverTap", isEnabled: false)
```